payment_manual/indico_payment_manual/__init__.py

```diff
@@ -16,6 +16,17 @@
 
 from __future__ import unicode_literals
 
+from indico.core import signals
 from indico.util.i18n import make_bound_gettext
 
 _ = make_bound_gettext('payment_manual')
+
+@signals.get_placeholders.connect_via('event-payment-form')
+def _get_payment_form_placeholders(sender, event, registration, **kwargs):
+    from indico_payment_manual.placeholders.payments import (IDPlaceholder, EventIDPlaceholder,
```

Comment:
I'd use `indico_payment_manual.placeholders` - no need to have a subpackage for placeholders. The plugin will only have one type anyway.